source/docs/3/landuse_buffer.rst

```diff
@@ -0,0 +1,235 @@
+
+   .. image:: /static/3/landuse_buffer/images/15.png
+      :align: center
+	  
+16. In the Buffer dialog, select LandUse2020 as the :guilabel:`Input layer`. Check the :guilabel:`Selected features only` box so the buffer will apply only to the selected institutional land use parcels. We will create a buffer zone of 100ft around each parcel. Enter 100 feet as the :guilabel:`Distance`. The default buffers are of circular shape. Since we are dealing with parcel data, a rectangular buffer area is more appropriate where each edge of the parcel will be offset by the buffer distance. Select ``Square`` as the :guilabel:`End cap style` and ``Miter`` as the :guilabel:`Join style`. Click the browse button next to Buffered and select :guilabel:`Save to File`.
+
+   .. image:: /static/3/landuse_buffer/images/16.png
+      :align: center
+	  
+17. Enter the output file name as ``buffer.shp`` and click OK. Click :guilabel:`Run`.
+	  
+   .. image:: /static/3/landuse_buffer/images/17.png
+      :align: center
+	  
+18. Once the processing finishes, a new layer buffer will be added to QGIS. These polygons represent the buffer zone of restrictions around the institutional land use.
+
+   .. image:: /static/3/landuse_buffer/images/18.png
+      :align: center
+	  
+
+19. We now know which areas of the city fall under the restricted zone. It would be useful to identify all parcels which fall in this zone and add an attribute indicating that a restriction applies to that parcel. Let’s first add a column to the buffer polygons. Search for :menuselection:`Field Calculator` in the processing toolbox. Double-click to open.
+
+   .. image:: /static/3/landuse_buffer/images/19.png
+      :align: center
+	  
+20. In the Field Calculator dialog, ensure buffer is selected as the :guilabel:`Input layer`. We don’t need to save this layer, so leave the Output file field empty. Enter ``RESTRICTED`` as the result :guilabel:`Field Name`. Set the :guilabel:`Result field type` to ``String``. Type ``Yes`` as the :guilabel:`Expression`. Click OK.
+
+   .. image:: /static/3/landuse_buffer/images/20.png
+      :align: center
+
+21. A new layer called Calculated will be added to QGIS. Open the Attribute Table and verify that a new column called RESTRICTED with value Yes exists in this layer.
+
+   .. image:: /static/3/landuse_buffer/images/21.png
+      :align: center
+	  
+22. Next step is to do a Spatial Join to add this attribute to the original parcels layer based on which parcels intersect with the buffer zone. The ``LandUse2020`` layer has more than 100000 elements. We will add spatial indexing to the features to improve the performance and speedup the spatial join operation. 
+Check out the course `Spatial Indexing <https://courses.spatialthoughts.com/advanced-qgis.html#concept-spatial-indexing>`_ section to understand more about this. 
```

Comment:
"Check out the section on `Spatial Indexing `Spatial Indexing <https://courses.spatialthoughts.com/advanced-qgis.html#concept-spatial-indexing>`_ from our Advanced QGIS course.

Reply:
DOne